setup/setup.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/sh
 
+PRESERVECONFIG=0
+if [ -f /opt/traccar/conf/traccar.xml ]
+then
+    cp /opt/traccar/conf/traccar.xml /opt/traccar/conf/traccar.xml.saved
+    PRESERVECONFIG=1
+fi
+
 mkdir -p /opt/traccar
 cp -r * /opt/traccar
 chmod -R go+rX /opt/traccar
 
+if [ ${PRESERVECONFIG} -eq 1 ] && [ -f /opt/traccar/conf/traccar.xml.saved ]
+then
+    mv /opt/traccar/conf/traccar.xml.saved /opt/traccar/conf/traccar.xml
```

Comment:
One other thing I just thought about. What if `mv` asks for confirmation before replacing the file? Is it possible?

Reply:
Many distributions have aliases, but the correct switch would be "-f"

Reply:
According to man page of mv:

`-f, --force
              do not prompt before overwriting`

`-i, --interactive
              prompt before overwrite`

In my experience just plain 'mv' never asks for permission. If we want to circumvent this then we should use full path to mv to exclude alias using. e.g:  /bin/mv.  Or add a 'rm' to remove any existing file first when it exists. This will keep alias usage in place.

So there are different solutions but as said in my experience the current has never asked questions and a default Ubuntu or Fedora installation.

Reply:
Is there any reason we can't use `mv -f`?

Reply:
No. I will add it an push the change